### PR TITLE
TabController - design tokens - fixes

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
TabController - design tokens - fixes

## Changelog
TabController - design tokens - fixes

### CODE REVIEW

File: src/components/tabController/TabBar.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ interface Props extends TabControllerBarProps, BaseComponentInjectedProps, Forwa
   children?: ChildProps[] | ChildProps;
 }
 
-const FADER_PROPS = {size: 76, tintColor: Colors.$backgroundDefault};
+const FADER_PROPS = {size: 76, tintColor: Colors.$backgroundElevated};
```
Comment (lidord-wix) [inline src/components/tabController/TabBar.tsx:120]: Are you sure about it?
I used the `backgroundElevated` originally and you said it looks better with `backgroundDefault`

**Review by lidord-wix — COMMENTED** [review]
Small comment

File: src/components/tabController/TabBar.tsx (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/tabController/TabBar.tsx:120]: It'll be `$backgroundElevated` but will be handled within the `Fader`

Comment (lidord-wix) [reply src/components/tabController/TabBar.tsx]: Are you sure that the default fader color should be `$backgroundElevated` and not `$backgroundDefault`? 
maybe it should be `$backgroundDefault` and only here we should override it with `$backgroundElevated`..

Comment (M-i-k-e-l) [reply src/components/tabController/TabBar.tsx]: That's what Yulia said in the relevant channel (can't link here).
In any case, if there will be errors we can fix them later.

Comment (M-i-k-e-l) [reply src/components/tabController/TabBar.tsx:124]: Changed

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**